recipes/cairo/all/conanfile.py

```diff
@@ -90,23 +103,18 @@ def requirements(self):
     def build_requirements(self):
         if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
             self.build_requires("msys2/cci.latest")
-        if not self._is_msvc:
+        if not microsoft.is_msvc(self):
             self.build_requires("libtool/2.4.6")
             self.build_requires("pkgconf/1.7.4")
             self.build_requires("gtk-doc-stub/cci.20181216")
```

Comment:
```suggestion
            self.tool_requires("libtool/2.4.6")
            self.tool_requires("pkgconf/1.7.4")
            self.tool_requires("gtk-doc-stub/cci.20181216")
```
https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#requirements